sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -139,28 +132,78 @@ namespace Azure { namespace Security { namespace Attestation {
      *
      * @note TPM attestation requires an authenticated attestation client.
      */
-    explicit AttestationClient(
```

Comment:
why is explicit gone?  We should not allow implicit conversions,

Reply:
What value does explicit have on a private constructor?